### DISCUSSION TITLE
Rest API with dynamic order by?

### DISCUSSION THREAD

**Discussion by kabouras** [hasura/graphql-engine#7763] [Question]
Hello, I wanted to clarify if it was possible to use dynamic "order by" clauses over the REST API. 

The where clause with "_bool_exp" works fine. Using "_order_by" returns the error message below. 

This is on v2.0.9.

**Query**
```
query myQuery($where: example_table_bool_exp, $order_by: [example_table_order_by!]) {
    example_table(where:$where, order_by: $order_by ) {
        name_field
        value_field
    }    
}
```

**Payload**
```
{

	"where": {
		"name_field": {
			"_eq": "Test"
		}
	},
	"order_by": [{
		"value_field": "asc"
	}]
}
```

**Response**
```
{
	"path": "$",
	"error": "The variable type for the expected variable order_by was not supported.",
	"code": "bad-request"
}
```

Does this mean "_order_by" is not supported by the RESTified API?  If not, will it be in the future? Suggested alternatives?

Many thanks!

**Comment by sordina** [comment]
Hi @kabouras This is a bug. The API should support list types but we have inadvertently let them fall through to an error case on the backend. We will ship a fix for this as soon as it has been addressed.
